contributing/labeling.md

```diff
@@ -0,0 +1,15 @@
+# Label Definitions
+
+* `Epic`: Big issue to be broken down into smaller tasks
+
+### Priority (`p.*`)
+* `p.Critical`: To be handled as soon as possible, usually a bug or security loophole
+* `p.High`: Issues corresponding to the upcoming `Milestone`
+* `p.Medium`: Issues corresponding to a future `Milestone`
+* `p.Low`: Good-to-have features not in the roadmap
+
+### Difficulty (`d.*`)
```

Comment:
Okay with collapsing intermediate and advanced but i was thinking `FirstTimer` issues would be very very simple changes, e.g. minor bugs, spelling errors, and `beginner` would be more like those in the PySyft repo at the moment. Many OSS projects have this distinction to help new contributors find easy issues to get started with.